.github/workflows/tox-tests.yml

```diff
@@ -88,3 +88,31 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: true
+  bandit-exitzero:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Maybe we should be using 3.8?

Reply:
The one (minor) risk could be Bandit fix suggestions which can't be implemented on an older Python version. Since there aren't any medium or higher findings currently, I don't see any reason not to go to 3.8 :)  I'll get it updated and changes pushed.